### PR TITLE
Registration

### PR DESCRIPTION
**Pull request by swapnith8**

### CODE REVIEW

File: application.py [hunk 1]
```diff
@@ -23,4 +23,10 @@
 
 @app.route("/")
 def index():
-    return "Project 1: TODO"
+    return render_template("Registration.html")
+@app.route("/Details",methods=["POST"])
```
Comment (ashishk144) [inline application.py]: change the route to /register and see that the function can handle both GET and POST methods

Comment (swapnith8) [reply application.py]: updated GET and POST method and also attached screenshot in the issue

Comment (ashishk144) [reply application.py]: okay work on next assignment.

**Review by ashishk144 — COMMENTED** [review]
Merge it and work on the ORM part.

File: app.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+@app.route("/")
+def index():
+    return render_template("Registration.html")
+@app.route("/register",methods=["POST","GET"])
+def cont():
+    db.create_all()
+    if request.method=="GET":
+        return ("Registration.html")
+    else:
+        userdata=userschema(request.form["username"],request.form["email"],request.form["city"],request.form["pwd"])
```
Comment (ashishk144) [inline app.py:43]: You can try encrypting and storing the password. If you've enough time, then only work on it, or continue on the next tasks.

**Review by ashishk144 — COMMENTED** [review]
Merge the branch and work on the authentication and raise a PR for the next one.

**Pull request merged**